cairo/mock_pragma/Scarb.toml

```diff
@@ -7,7 +7,7 @@ edition = "2023_10"
 
 [dependencies]
 starknet = "2.2.0"
-pragma = { git = "https://github.com/astraly-labs/pragma-oracle", tag = "v1.0.5" }
+pragma_lib = { git = "https://github.com/astraly-labs/pragma-lib", tag = "2.3.1" }
```

Comment:
is there a reason why we're using 2.3.1?

Reply:
It is the oldest version tagged in the `pragma_lib` - since it's compiling fine I kept it like this. Shall I create a new tag for an earlier version?

Reply:
I don't know, if this is the same behavior as production code you can keep it, otherwise we can bump. your call

Reply:
We can keep it like this then - does not change anything for the types/abis